src/models/responses.ts

```diff
@@ -0,0 +1,26 @@
+import { Device, Playlist, Song } from "./models";
+
+export interface SpotifyReponse {
```

Comment:
Create an overall Response object that I can base my other interfaces on. Sorta like SpotifyResponse is the superclass of all of my other interfaces here, except for the Error interface, which instead is a property of all SpotifyResponses. Wanted that error on there since all http responses could fail